docs/scenarios/manifests/README.md

```diff
@@ -0,0 +1,65 @@
+## Generating manifests using CPMA and applying them
+
+CPMA analyzes OCP3 configuration files and when possible transforms them to CRs, generated CRs are used to configure OCP4 cluster as `day two` operations.
+
+---
+
+
+### 1. Prerequisites
+
+Prior to working with CPMA you need to deploy an OCP 3.7+ cluster(OCP 3.7, 3.9, 3.10, 3.11 will be supported).
+There are to 2 ways for cpma to obtain desired configurations:
+
+1. SSH, CPMA is capable to connect to cluster using SSH and fetch configurations to a local directory. For using SSH it's required to have a proper private key and SSH user may require sudo permissions for reading configurations.
+
+2. Configurations stored in local directory, it's possible to manualy download configuration and feed them to CPMA.
```

Comment:
Similarly:
`2. Locally stored configuration files. If option 1 is no possible, files can be retrieved prior to executing CPMA. The configuration files are expected to be located in '<workdir>' parameter and under the FQDN target hostname.`